### PR TITLE
Adding support to mMARCO v2

### PR DESCRIPTION
**Pull request by lhbonifacio**
We are adding support for [mMARCO v2](https://github.com/unicamp-dl/mMARCO).

### CODE REVIEW

**Review by rodrigonogueira4 — APPROVED** [review]
LGTM! @lintool, this PR is because we now have newer models (v2) fine-tuned on mMARCO.

**Comment by lintool** [comment]
@rodrigonogueira4 this looks fine to me... @ronakice any comments?

**Review by ronakice — APPROVED** [review]
LGTM!

File: pygaggle/rerank/transformer.py [hunk 1]
```diff
@@ -32,17 +32,23 @@
...
-        'unicamp-dl/ptt5-base-msmarco-en-pt-10k':  ['▁não'  , '▁sim'],
-        'unicamp-dl/mt5-base-multi-msmarco':       ['▁no'   , '▁yes'],
-        'unicamp-dl/mt5-base-en-pt-msmarco':       ['▁no'   , '▁yes']
+        'castorini/monot5-base-msmarco':             ['▁false', '▁true'],
+        'castorini/monot5-base-msmarco-10k':         ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco':            ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco-10k':        ['▁false', '▁true'],
+        'castorini/monot5-base-med-msmarco':         ['▁false', '▁true'],
+        'castorini/monot5-3b-med-msmarco':           ['▁false', '▁true'],
+        'unicamp-dl/mt5-base-en-msmarco',            ['▁no'   , '▁yes'],
```
Comment (rodrigonogueira4) [inline pygaggle/rerank/transformer.py:41]: @lhbonifacio , could you please fix this? It should be `:` instead of `,` after the model name

**Pull request merged**